pkg/upgrade/key_pair.go

```diff
@@ -8,42 +8,55 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
-	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/cloud/aws/services/certificates"
-	clusterapiv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	clusterv1alpha1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 )
 
-// KeyPair contains a cert and key.
-type KeyPair struct {
+// KubeconfigSecretKey is the key where the kubeconfig is stored in the secret.
+const KubeconfigSecretKey = "kubeconfig"
```

Comment:
unexport?

Reply:
yeah, that's fine. The purpose of this is to make it obvious what the secret key is and that purpose is still served with an unexported const 👍